### DISCUSSION TITLE
Codewars keeps showing me only completed Katas when I click the 'Next Kata" button.

### DISCUSSION THREAD

**Discussion by ThomasAurelius** [codewars/codewars.com#2650] [Q&A]
I keep having to go to practice, and filter to find ones that I havent done. How do I fix this?

**Comment by Blind4Basics** [comment]
what's the current setup in the dashboard, for the kata suggestsions? What you choose there for the second drop down (the "focus" one) applies when you click on next kata, I believe.

https://docs.codewars.com/getting-started/finding-kata/#personal-trainer
